### PR TITLE
Trailing widget override for ExpansionTile

### LINKED ISSUE #11890: ExpansionTile optional trailing icon
ExpansionTile has a trailing icon that rotates when expanded.
Unfortunately, I need this space for some other functionality.
It would be nice if this trailing icon could be optional, and the expansion was only triggered by a click elsewhere on the tile.

### PR DESCRIPTION
**Pull request by Skylled**
Fixes #11890

### CODE REVIEW

File: packages/flutter/lib/src/material/expansion_tile.dart [hunk 1]
```diff
@@ -38,6 +38,7 @@ class ExpansionTile extends StatefulWidget {
     this.backgroundColor,
     this.onExpansionChanged,
     this.children: const <Widget>[],
+    this.trailing
```
Comment (Hixie) [inline packages/flutter/lib/src/material/expansion_tile.dart]: nit: trailing comma

**Comment by Hixie** [comment]
Needs a test.

**Comment by Skylled** [comment]
While I'm poking around in here, is there a reason `two_level_list_test.dart` hasn't been renamed to `expansion_tile_test.dart`?

**Comment by Hixie** [comment]
No _good_ reason...

**Comment by Hixie** [comment]
Test looks great, thanks!

**Comment by Hixie** [comment]
LGTM
Thanks for your contribution! I will land this now that the tree is calm.

**Pull request merged**